### PR TITLE
add examples with contribution instructions

### PR DESCRIPTION
**Pull request by NTCoding**
I've add an example core domain pattern and an invitation for other people to contribute their patterns (already received 1 request).

### CODE REVIEW

File: examples/first-to-market-core.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+# First-to-Market Core
+
+![First to  market core](/resources/first-to-market-core.png)
+
+A First-to-market Core provides an advantage to the organisation which develops the capability first. But since the complexity of the feature is low, the competition will soon catch up and it will not be the core domain for a long period of time.
+
```
Comment (yellowbrickc) [inline examples/first-to-market-core.md:6]: A very cool description of the problem but I think it is not really an example, rather an explanation. What about adding a real-life story like 
```suggestion
Let say, you have a product like Meetup but yo have some great ideas about how to beat them in the long run. You know that your core business is scheduling and managing events but this is only your home-work. Soon after this is in place you will focus on the unique selling point for your product, like supporting different remote working tools (?).
```

What do you think? Maybe you have a better example, it is not so easy to come up with the 5-billion product idea in 2 minutes :D

File: README.md [hunk 2]
```diff
@@ -22,6 +22,10 @@ With a slight tweak of the y-axis label, core domain charts can be used to plan
 
 ![alt text](resources/architecture-migration-core-domain-chart.jpg "Architecture Migration Core Domain Chart")
 
+## Examples
+
+Check out the [Examples](/examples). Please feel free to create a pull request with your own suggestions.
```
Comment (yellowbrickc) [inline README.md:27]:
```suggestion
Check out the [Examples](/examples) to get a better understanding of these charts. 

Own examples via a pull request are very welcome!
```

Comment (NTCoding) [reply README.md]: Thanks.

File: examples/first-to-market-core.md (hunk 1 above)
Comment (NTCoding) [reply examples/first-to-market-core.md:6]: Yes, this is very important to examples. Thanks.

**Pull request merged**